org.speed_dreams.SpeedDreams.yaml

```diff
@@ -72,14 +74,14 @@ modules:
       - CXXFLAGS=-fPIC
     sources:
       - type: archive
-        url: http://plib.sourceforge.net/dist/plib-1.8.5.tar.gz
+        url: https://plib.sourceforge.net/dist/plib-1.8.5.tar.gz

```

Comment:
I get connection refused on this.

Could be another sourceforget hiccup.

Reply:
The website of the library does not have a secure connection, it is an old website, in fact it is a library that has been abandoned for years. I have looked at SourceForge and the only file there is from a release candidate of this version (1.8.5-rc1):
https://sourceforge.net/projects/plib/files/plib/

Reply:
Yeah, I think I added https after building and didn't notice. I was going to say this but I'm compiling it again to make sure I didn't make another mistake.